crates/bevy_asset/src/asset_server.rs

```diff
@@ -567,38 +600,39 @@ mod test {
                 handle_to_path: Default::default(),
                 asset_lifecycles: Default::default(),
                 task_pool: Default::default(),
-                asset_io: Box::new(FileAssetIo::new(&".")),
+                asset_io: Box::new(FileAssetIo::new(asset_path)),
             }),
         };
         asset_server.add_loader::<FakePngLoader>(FakePngLoader);
```

Comment:
I think each test should manually opt in to relevant loaders. This makes each test more self-documenting, removes redundant work (ex no need to add FailingLoader to the `extensions()` test) and ensures that `setup` can be used in multiple contexts (ex: maybe we eventually want a test to have a FailingFakePngLoader, but that will conflict with FakePngLoader).

Reply:
Yep totally agree!
I just removed them and manually added them to each test case.

Reply:
Just got the failing tests to work again, so all should be good :)